client/src/Components/UserList/reducer.js

```diff
@@ -0,0 +1,22 @@
+import { LOADING, FOUND_USERS, ERROR } from "./actions";
```

Comment:
We probably want to have a consistent naming convention across our actions. Ideally, we'd want to have something specific (since we can see it in the Redux devtools). The convention that I've typically seen has been
```
VERB_ITEM_STATUS
Examples:
FETCH_USERS_ATTEMPT (or started?)
UPDATE_USER_SUCCESS
DELETE_PROPERTY_FAILURE (or error?)
```

Reply:
Changed naming in reducer and action file.